runtime/examples/CMakeLists.txt

```diff
@@ -13,4 +13,26 @@ link_directories(${CMAKE_CURRENT_SOURCE_DIR}/lib)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
 
 add_executable(a.out run.cpp)
-target_link_libraries(a.out blueoil pthread)
+
+if(USE_OPENCV)
+  find_package( OpenCV REQUIRED )
```

Comment:
Could you remove the spaces next to the parenthesis?

Reply:
https://github.com/blue-oil/blueoil/pull/577/commits/506e97a20f1be59e596f4855a61ab3e2c4cd2196